robottelo/ui/browser.py

```diff
@@ -173,6 +176,8 @@ def _create_container(self):
             host_config=self._client.create_host_config(
                 publish_all_ports=True),
             image='selenium/standalone-firefox',
+            name=self._name.split('.', 4)[-1] + '_{0}'.format(
```

Comment:
If name is None then this will raise an exception. Better to check if the name is set before calling split.